internal/handlers/common/params.go

```diff
@@ -70,6 +70,8 @@ func AssertType[T types.Type](value any) (T, error) {
 var (
 	errUnexpectedType = fmt.Errorf("GetWholeNumberParam: unexpected type")
 	errNotWholeNumber = fmt.Errorf("GetWholeNumberParam: not a whole number")
+	errNegativeNumber = fmt.Errorf("getBinaryMaskParam: negative number")
```

Comment:
could some context be added to the error? i.e. that it's for bitwise op

Reply:
It might be used not just for bitwise operators.

Reply:
ok